pkg/tsdbctl/delete.go

```diff
@@ -111,9 +111,13 @@ func (dc *delCommandeer) delete() error {
 		}
 	}
 
+	partialMsg := ""
+	if !dc.deleteAll {
+		partialMsg = " part of"
+	}
 	if !dc.force {
 		confirmedByUser, err := getConfirmation(
-			fmt.Sprintf("You are about to delete TSDB table '%s' in container '%s'. Are you sure?", dc.rootCommandeer.v3iocfg.TablePath, dc.rootCommandeer.v3iocfg.Container))
+			fmt.Sprintf("You are about to delete%s TSDB table '%s' in container '%s'. Are you sure?", partialMsg, dc.rootCommandeer.v3iocfg.TablePath, dc.rootCommandeer.v3iocfg.Container))
```

Comment:
add space before %s and remove from the message

Reply:
it won't work. the %s can be empty string as well, so I don't want 2 spaces there

Reply:
Just update the message and you'll be able to do what you want.
For example:
```
if !dc.deleteAll {
	partialMsg = " part of"
} else {
	partialMsg = "entire"
}
```
Note, in some cases the given time interval could be greater than your actual database range, so saying "partial" in any case is incorrect.